.storybook/env.js

```diff
@@ -1,3 +1,4 @@
-export const CLIENTID = 'ac77046c-156c-40f0-8507-3b5a58034582';
+export const CLIENTID = '6b5a42f1-9e1a-4a20-9df2-9b058df3c213';
```

Comment:
Let's hold this change until tomorrow when I get some time with Michael to set up a new AAD app for this